### PR TITLE
Multiple container

### PR DESCRIPTION
**Pull request by Bennables**
This is the code that can make Hana run. Right now only works with Springtrap.

### CODE REVIEW

File: src/main/java/frc/robot/swerve/SwerveSteerController.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public SwerveSteerController(
     config.sensorDirection = false;
     config.initializationStrategy = SensorInitializationStrategy.BootToAbsolutePosition;
 
-    encoder = new CANCoder(canCoderPort);
+    encoder = new CANCoder(canCoderPort,"CAN_Network");
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/swerve/SwerveSteerController.java]: Should we use the `canCoderPort`?

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -62,36 +62,8 @@ public static final class DrivetrainConstants {
...
-    // public static final double BACK_LEFT_MODULE_STEER_OFFSET = -toRadians(270);
-    public static final double BACK_LEFT_MODULE_STEER_OFFSET = -toRadians(315);           //315.17578125
-
-    public static final int BACK_RIGHT_MODULE_DRIVE_MOTOR = 5;
-    public static final int BACK_RIGHT_MODULE_STEER_MOTOR = 6;
-    public static final int BACK_RIGHT_MODULE_STEER_ENCODER = 13;
-    // public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -toRadians(225);
-    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -toRadians(340.5-180);         //163.125
+    //"rio" for springtrap, "CAN_Network" for hana
+    public static final String CANIVORE_NAME = "CAN_Network";
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/Constants.java:66]: Should this be part of DriveTrainConstants class?

**Review by RobertTheBlair — CHANGES_REQUESTED** [review]
Please figure out if we need to use the port name in SwerveSteerController

File: src/main/java/frc/robot/swerve/SwerveSteerController.java (hunk 1 above)
Comment (RobertTheBlair) [reply src/main/java/frc/robot/swerve/SwerveSteerController.java]: Or somehow pass in the network name

Comment (Bennables) [reply src/main/java/frc/robot/swerve/SwerveSteerController.java]: I'll do it!

**Pull request merged**